docs/scripts.md

```diff
@@ -22,53 +28,7 @@ const dirName = 'foo bar';
-
-### Flexibility
-
-Unlike shell languages like Bash, libraries like Execa and zx enable you to write scripts with a more featureful programming language (JavaScript). This allows complex logic (such as [parallel execution](#parallel-commands)) to be expressed easily. This also lets you use [any Node.js package](#builtin-utilities).
-
-### Shell
-
-The main difference between Execa and zx is that Execa does not require any shell. Shell-specific keywords and features are [written in JavaScript](#variable-substitution) instead.
-
-This is more cross-platform. For example, your code works the same on Windows machines without Bash installed.
-
-Also, there is no shell syntax to remember: everything is just plain JavaScript.
-
-If you really need a shell though, the [`shell` option](../readme.md#shell) can be used.
-
-### Simplicity
-
-Execa's scripting API mostly consists of only two methods: [`` $`command` ``](../readme.md#command) and [`$(options)`](../readme.md#options).
-
-[No special binary](#main-binary) is recommended, no [global variable](#global-variables) is injected: scripts are regular Node.js files.
-
-Execa is a thin wrapper around the core Node.js [`child_process` module](https://nodejs.org/api/child_process.html). Unlike zx, it lets you use [any of its native features](#background-processes): [`pid`](#pid), [IPC](https://nodejs.org/api/child_process.html#subprocesssendmessage-sendhandle-options-callback), [`unref()`](https://nodejs.org/api/child_process.html#subprocessunref), [`detached`](https://nodejs.org/api/child_process.html#child_processspawncommand-args-options), [`uid`](https://nodejs.org/api/child_process.html#child_processspawncommand-args-options), [`gid`](https://nodejs.org/api/child_process.html#child_processspawncommand-args-options), [`cancelSignal`](https://nodejs.org/api/child_process.html#child_processspawncommand-args-options), etc.
-
-### Modularity
-
-zx includes many builtin utilities: `fetch()`, `question()`, `sleep()`, `stdin()`, `retry()`, `spinner()`, `chalk`, `fs-extra`, `os`, `path`, `globby`, `yaml`, `minimist`, `which`, Markdown scripts, remote scripts.
-
-Execa does not include [any utility](#builtin-utilities): it focuses on being small and modular instead. Any Node.js package can be used in your scripts.
-
-### Performance
-
-Spawning a shell for every command comes at a performance cost, which Execa avoids.
-
-Also, [local binaries](#local-binaries) can be directly executed without using `npx`.
-
-### Debugging
-
-Child processes can be hard to debug, which is why Execa includes a [`verbose` option](#verbose-mode).
-
-Also, Execa's error messages and [properties](#errors) are very detailed to make it clear to determine why a process failed.
-
-Finally, unlike Bash and zx, which are stateful (options, current directory, etc.), Execa is [purely functional](#current-directory), which also helps with debugging.
-
-## Details
+## Examples
```

Comment:
Move this to the bottom. Users prefer to read code first, then read the long explanation later.